django/website/logframe/templates/logframe/create_logframe.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+    <h2>{% trans "Create a new logframe" %}</h2>
```

Comment:
Inconsistent capitalisation of "logframe". I think it has a capital L elsewhere.